src/doveadm/doveadm-mail.c

```diff
@@ -1021,6 +1021,17 @@ doveadm_cmd_ver2_to_mail_cmd_wrapper(struct doveadm_cmd_context *cctx)
 					wildcard_user = arg->value.v_string;
 					mctx->cur_username = NULL;
 				}
+			} else if (strchr(arg->value.v_string, ',') != NULL) {
+				mctx->add_username_header = TRUE;
+				if (!is_client) {
+					char *users = p_str_replace(mctx->pool,
```

Comment:
this should be done using `t_strsplit_tabescaped` and counterpart on sending should use `str_append_tabescaped` and data should be tab separated. you might need to enclose it twice.

Reply:
Not sure I understand. This is already the server/worker side. I'm reusing the same mechanism the `-F` commandline argument is using which expects the data to be LF terminated.

The wire data is like this:
```
C: \tsomeuser\tquota get\t-u\tmailbox1,mailbox3\n
S: mailbox1\tUser quota\t[...]\tmailbox3\tUser quota\t[...]\n+\n
```

`t_strsplit_tabescaped` splits per tab but I have commas.

Reply:
there is no reason why the username cannot contain tabs so it needs to be tabescaped. username can also contain a comma, so it cannot use comma as separator, it should use tab as separator. so what you need to do is to make the list with tab escaped values, separated by tabs, and run the list again over str_tabescaped. then you need to de-escape it twice on the receiving side.

Reply:
Alright. Will see what I can come up with.

> username can also contain a comma, so it cannot use comma as separator

Yeah, I know. But it was the easiest way forward and wanted to see if you are interested at all. Maybe I should ditch the `-u` argument and use `-F\t<list>` on the wire. Sending over a path to a local file isn't of much use anyway.